stellarobservatory/quorums.py

```diff
@@ -0,0 +1,53 @@
+"""Torstens's quorum enumeration"""
+
+
+from typing import Callable
+from typing import AbstractSet
+from typing import Type
+
+# Allow for defining an FBAS as a function: (set<T>, T) -> bool.
+# This function returns True, iff the FBAS has a slice for the given node T in the given
+# set.
+FBAS = Callable[[AbstractSet[Type], Type], bool]
+
+def enumerate_quorums(fbas: FBAS, all_nodes):
```

Comment:
Do we want to call it `is_slice_contained` instead of `fbas`?

Reply:
Sure, same with the type definition above? `FBAS`. That thing only exists as some form od documentation what `fbas` (or now  `is_slice_contained`) should look like (a type hint in python lingo).

Reply:
The type FBAS now is a pair:
```python
IsSliceContained = Callable[[AbstractSet[Type], Type], bool]
FBAS = (IsSliceContained, AbstractSet[Type])
```
In Torsten's code this was the other way around fbas = (V, contains_slice).